misk/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -1,12 +1,23 @@
 package misk.hibernate
 
 import com.google.common.util.concurrent.Service
-import com.google.inject.AbstractModule
+import misk.inject.KAbstractModule
 import misk.inject.addMultibinderBinding
-import misk.inject.to
+import misk.jdbc.DataSourceConfig
+import org.hibernate.SessionFactory
+import javax.inject.Provider
+import kotlin.reflect.KClass
 
-class HibernateModule : AbstractModule() {
+class HibernateModule(
+  private val qualifier: KClass<out Annotation>,
+  private val config: DataSourceConfig,
+  private val entityClasses: Set<KClass<*>>
```

Comment:
you probably want to rebase off joey's latest JPAEntityModule which allows composing multiple sets of entities into a single set.

Reply:
Will address in follow-up.